.gitignore

```diff
@@ -24,5 +24,6 @@ private.properties
 .gradle/
 .idea/libraries/
 main/build/
+main/proguard/
```

Comment:
Where do those files come from? I don't see anything like that in my folders. What is your environment/build-process?